hazelcast/src/main/java/com/hazelcast/cp/event/impl/CPGroupAvailabilityEventImpl.java

```diff
@@ -73,7 +74,9 @@ public int getMajority() {
 
     @Override
     public boolean isMajorityAvailable() {
-        return missingMembers.size() < getMajority();
+        HashSet<CPMember> leftMembers = new HashSet<>(members);
```

Comment:
Minor `Set<CPMember>`

Reply:
I agree if this were on the API but it's implementation detail so I think concrete implementation is fine.